Applications/CLI/ogs.cpp

```diff
@@ -144,6 +145,11 @@ int main(int argc, char *argv[])
                 vtkSmartPointer<vtkMPIController>::New();
             controller->Initialize(&argc, &argv, 1);
             vtkMPIController::SetGlobalController(controller);
+
+            logog_setup.setFormatter(
+                std::make_unique<BaseLib::TemplateLogogFormatterSuppressedGCC<
+                    TOPIC_LEVEL_FLAG | TOPIC_FILE_NAME_FLAG |
+                    TOPIC_LINE_NUMBER_FLAG>>());
```

Comment:
Will that automatically nicely format the `[mpi_rank]` line prefixes?

Reply:
The output looks like:
```
[9] Assemble HTProcess.
[1] Assemble HTProcess.
[3] Assemble HTProcess.
[0] Assemble HTProcess.
```